backend/fms_core/models/run_type.py

```diff
@@ -3,16 +3,16 @@
 from django.core.exceptions import ValidationError
 from django.db import models
 
-from fms_core.models import Protocol
+from fms_core.models import Protocol, Platform
 from .tracked_model import TrackedModel
 
 from ._constants import STANDARD_NAME_FIELD_LENGTH
 from ._validators import name_validator
 from ._utils import add_error as _add_error
 
-__all__ = ["ExperimentType"]
+__all__ = ["RunType"]
 
-PROTOCOLS_BY_EXPERIMENT_TYPE_NAME = {
+PROTOCOLS_BY_RUN_TYPE_NAME = {
```

Comment:
Same comment as previously, didnt you get rid of dictionaries with Protocol list to have that stored in the db?